libs/withdraws/src/lib/withdrawal-dialogs.tsx

```diff
@@ -24,13 +26,16 @@ export const WithdrawalDialogs = ({
         onChange={(isOpen) => setWithdrawDialog(isOpen)}
         size="small"
       >
-        <WithdrawFormContainer
-          partyId={keypair?.pub}
-          submit={(args) => {
-            setWithdrawDialog(false);
-            createWithdraw.submit(args);
-          }}
-        />
+        {withdrawDialog && (
```

Comment:
question (blocking): Why is this condition required?

Reply:
not anymore